pkg/framework/machinesets.go

```diff
@@ -153,7 +160,7 @@ func GetWorkerMachineSets(client runtimeclient.Client) ([]*mapiv1beta1.MachineSe
 	// but the Machines themselves are labled as such via the template., so we
 	// can reach into the template and check the lables there.
 	for i, ms := range machineSets.Items {
-		labels := ms.Spec.Template.GetLabels()
```

Comment:
Aren't those two lines functionally equal?..

Reply:
They are normally, except we changed the type of the `ObjectMeta` to our own version which doesn't have the accesor methods defined, so we have to do this now